packages/tracing/src/browser/browsertracing.ts

```diff
@@ -40,6 +40,14 @@ export interface BrowserTracingOptions extends RequestInstrumentationOptions {
    */
   finalTimeout: number;
 
+  /**
+   * The heartbeat interval. If activities don't change in 3 heartbeats, a transaction will be finished.
+   * Time is in ms.
```

Comment:
I'd rephrase this a little because I think that most users won't know what "activities" are

```suggestion
   * The heartbeat interval. If no new spans are started or open spans are finished within 3 heartbeats, 
   * the transaction will be finished. 
   * Time is in ms.
```